benchmarks/common.py

```diff
@@ -1401,6 +1401,12 @@ def parse_args():
     parser.add_argument(
         "--prims-nvfuser", action="store_true", help="user prims + nvfuser backend"
     )
+    parser.add_argument(
```

Comment:
Nit - We can directly use `--backend=nvprims_nvfuser`, if you didn't know about this option.

Reply:
I didn't know! I will try and remove this code then.